app/controllers/events_controller.rb

```diff
@@ -10,7 +10,7 @@ def participants
       { label: @event.name, url: event_path(@event) },
       { label: 'participants' }
     ]
-    @requests = @event.travel_sponsorships.includes(:user).select(:user_id).distinct.accessible_by(current_ability)
+    @requests = @event.travel_sponsorships.includes(:user).group(:user_id).accessible_by(current_ability)
```

Comment:
In the previous query we are only selecting the user_id from a request for distinct requests if we also select state here then the requests will not be distinct, also group returns the latest request of the user(active request) I don't know if distinct also do the same.